Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -26,7 +26,30 @@
 USE_SSL: bool = not PARAMS.get('insecure', False)
 APP: Flask = Flask('demisto-teams')
 PLAYGROUND_INVESTIGATION_TYPE: int = 9
-GRAPH_BASE_URL: str = 'https://graph.microsoft.com'
+ENDPOINT = PARAMS.get('endpoint', 'Default - Worldwide (.com)')
+LOGIN_ENDPOINTS = {
```

Comment:
please move those consts to the api module

Reply:
The changes I did are for the Teams integration (which doesn't use the api module) and not for the TeamsManagemet, so I can't move it there